build.sbt

```diff
@@ -63,29 +63,29 @@ val googleApiServicesPubsubVersion = s"v1-rev20220904-$googleClientsVersion"
 val googleApiServicesStorageVersion = s"v1-rev20220705-$googleClientsVersion"
 
 // check versions from libraries-bom
-// https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.11.0/index.html
-val animalSnifferAnnotationsVersion = "1.22"
-val bigQueryStorageBetaVersion = "0.158.1"
-val bigQueryStorageVersion = "2.34.1"
+// https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.14.0/index.html
+val animalSnifferAnnotationsVersion = "1.23"
+val bigQueryStorageBetaVersion = "0.160.1"
+val bigQueryStorageVersion = "2.36.1"
 val checkerFrameworkVersion = "3.32.0"
 val errorProneAnnotationsVersion = "2.18.0"
 val failureAccessVersion = "1.0.1"
 val floggerVersion = "0.7.4"
-val gaxHttpJsonVersion = "0.108.0"
-val gaxVersion = "2.23.3"
-val googleApiCommonVersion = "2.6.3"
+val gaxHttpJsonVersion = "0.111.0"
+val gaxVersion = "2.26.0"
+val googleApiCommonVersion = "2.9.0"
 val googleAuthVersion = "1.16.0"
-val googleCloudBigTableVersion = "2.20.1"
-val googleCloudCoreVersion = "2.13.0"
-val googleCloudDatastoreVersion = "0.105.1"
-val googleCloudMonitoringVersion = "3.14.0"
-val googleCloudPubSubVersion = "1.105.7"
-val googleCloudSpannerVersion = "6.38.0"
-val googleCloudStorageVersion = "2.20.2"
-val googleCommonsProtoVersion = "2.14.3"
+val googleCloudBigTableVersion = "2.22.0"
+val googleCloudCoreVersion = "2.16.0"
+val googleCloudDatastoreVersion = "0.105.5"
```

Comment:
Not sure how these versions are selected/aligned, but just FYI:
the latest version available is `0.106.0`.
According to https://mvnrepository.com/artifact/com.google.api.grpc/proto-google-cloud-datastore-v1 .

Reply:
Versions are the one from google's BOM used by beam [here](https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.14.0/index.html). See top comment from the group